src/arg_parser.rs

```diff
@@ -42,20 +42,26 @@ where
                         .help("Transactions database file"),
                 )
                 .arg(
-                    Arg::with_name("bids_csv")
-                        .long("bids-csv")
+                    Arg::with_name("allocations_csv")
+                        .long("allocations-csv")
+                        .help("Input CSV contains token allocations, not bids"),
+                )
+                .arg(
+                    Arg::with_name("input_csv")
+                        .long("input_csv")
                         .required(true)
                         .takes_value(true)
                         .value_name("FILE")
-                        .help("Bids CSV file"),
+                        .help("Input CSV file"),
                 )
                 .arg(
                     Arg::with_name("dollars_per_sol")
                         .long("dollars-per-sol")
                         .required(true)
                         .takes_value(true)
                         .value_name("NUMBER")
-                        .help("Dollars per SOL"),
+                        .help("Dollars per SOL, if input CSV contains bids")
+                        .conflicts_with("allocations_csv")
```

Comment:
Cool feature, but no conflict. It's still useful for printing the fiat value of all SOL distributed.